### PR TITLE
adding binary package , as it is required for running th rag script

### PR DESCRIPTION
**Pull request by stkolanu**
## I got this issue when i tried to run the Local PDF code.
When i tried to follow the instruction, and run the script it mentioned me to have psycopg-binary package installed.
So, i tried adding it in the docs.
Let me know if its the right thing or not.
Getting to know things.

### CODE REVIEW

File: basics.mdx [hunk 1]
```diff
@@ -281,7 +281,7 @@ Let's test out RAG by building a **PDF Assistant** that helps us with food recip
     <CodeGroup>
 
     ```bash Mac
-    pip install -U pgvector pypdf psycopg sqlalchemy
+    pip install -U pgvector pypdf psycopg sqlalchemy psycopg-binary
```
Comment (ashpreetbedi) [inline basics.mdx:284]: @stkolanu thank you, i appreciate your contribution so much. Please replace `psycopg` with `psycopg-binary`.

Also please update the windows instructions on line 288.

Thank you again.

File: basics.mdx [hunk 2]
```diff
@@ -281,7 +281,7 @@ Let's test out RAG by building a **PDF Assistant** that helps us with food recip
...
     ```
 
     ```bash Windows
```
Comment (ashpreetbedi) [inline basics.mdx:287]: @stkolanu please replace `psycopg` with `psycopg-binary` here as well.

**Comment by ashpreetbedi** [comment]
@stkolanu thank you and yes, absolutely the right thing to do. Quick change to replace `psycopg` with  `psycopg-binary` otherwise it will install psycopg first

**Review by ashpreetbedi — CHANGES_REQUESTED** [review]

**Comment by ashpreetbedi** [comment]
@stkolanu mentioned you here: https://discord.com/channels/965734768803192842/965734768803192845/1205960434843127849

**Review by ashpreetbedi — APPROVED** [review]

**Comment by stkolanu** [comment]
How to merge this PR?

**Comment by ashpreetbedi** [comment]
Just merged :) thank you for contributing

**Pull request merged**